community/examples/slurm-gcp-v5-high-io.yaml

```diff
@@ -0,0 +1,135 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hpc-cluster-high-io-v5
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: high-io-slurm-gcp-v5
+  region: us-central1
+  zone: us-central1-c
+  enable_reconfigure: true
+
+# Documentation for each of the modules used below can be found at
+# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md
+
+deployment_groups:
+- group: primary
+  modules:
+  # Source is an embedded module, denoted by "modules/*" without ./, ../, /
+  # as a prefix. To refer to a local or community module, prefix with ./, ../ or /
+  # Example - ./modules/network/pre-existing-vpc
+  - id: network1
+    source: modules/network/vpc
+
+  - id: homefs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      local_mount: /home
+
+  - id: projectsfs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      filestore_tier: HIGH_SCALE_SSD
+      size_gb: 10240
+      local_mount: /projects
+
+  - id: scratchfs
+    source: community/modules/file-system/DDN-EXAScaler
+    use: [network1]
+    settings:
+      local_mount: /scratch
+
+  - id: low_cost_node_group_n2s2
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      name: n2s2
+      machine_type: n2-standard-2
+      node_count_dynamic_max: 10
+
+  - id: low_cost_node_group_n2s4
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      name: n2s4
+      machine_type: n2-standard-4
+      node_count_dynamic_max: 10
+
+  - id: low_cost_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use:
+    - network1
+    - homefs
+    - scratchfs
+    - projectsfs
+    - low_cost_node_group_n2s2
+    - low_cost_node_group_n2s4
+    settings:
+      is_default: true
+      partition_name: lowcost
+      enable_placement: false
+      exclusive: false
+
+  # This compute_partition is far more performant than low_cost_partition.
+  - id: compute_node_group_c2s60
```

Comment:
We might want to avoid mixing AMD and Intel cpus on the same partition.  
Also, we should provide a recommendation on how users are expected to direct jobs to a specific node group.  I would do it here with a one-liner and in more detail on the readme.

Reply:
Done

Reply:
Updated.
I only show how to specify the upper node group, since --mincpus only sets a minimum. I can also add a command with --nodelist if you prefer; it's documented more completely in the README.